packages/backend/README.md

```diff
@@ -25,7 +25,7 @@ You should only need to do this once.
 After that, go to the `packages/backend` directory and run
 
 ```bash
-AUTH_GOOGLE_CLIENT_ID=x AUTH_GOOGLE_CLIENT_SECRET=x SENTRY_TOKEN=x yarn start
+AUTH_GOOGLE_CLIENT_ID=a AUTH_GOOGLE_CLIENT_SECRET=b AUTH_GITHUB_CLIENT_ID=c AUTH_GITHUB_CLIENT_SECRET=d SENTRY_TOKEN=e LOG_LEVEL=debug yarn start
```

Comment:
keep them as `x`, to match the text below

Reply:
Fixed.